SimpleCalculator/src/com/tbd/sqt/Calculator.java

```diff
@@ -12,6 +12,22 @@ public Calculator (double num1, double num2) {
         number2 = num2;
     }
 
+    /*
+    Author: Rohini Bansal
+    ID: 500209290
+    Function : Addition Operation between 2 numbers
+    Arguments : user input numbers 2 double values
+    Result : one double value
+     */
+    public void add(double number1, double number2){
+        try{
+            result=number1+number2;
```

Comment:
Minor change -> Maybe we can add empty spaces to have a good format -> result = number1 + number2;

Reply:
implemented